<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>666</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="5">
     <widget class="QPushButton" name="PB_Quit">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="PB_File">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add File</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="PB_Clear">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" rowspan="3">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="CB_SKUSelect">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>OLxxx</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CM400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CFPxxx</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QPlainTextEdit" name="Text_drop">
      <property name="plainText">
       <string>Drop Files here</string>
      </property>
      <property name="overwriteMode">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3" rowspan="3" colspan="3">
     <widget class="QScrollArea" name="scrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>218</width>
         <height>459</height>
        </rect>
       </property>
       <widget class="QCheckBox" name="CB_SWversion">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>196</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Software version (WZ)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="CB_1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>138</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Heatsink (KN1)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="CB_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>184</y>
          <width>234</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Low Pressure Switch (SW1)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="CB_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>44</y>
          <width>138</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>AF NTC (KN2)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="CB_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>217</y>
          <width>237</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>High Pressure Switch (SW2)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="CB_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>295</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Probe1 NTC (KN4)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="CB_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>162</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Probe2 NTC (KN5)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="CB_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>77</y>
          <width>295</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>PC NTC (KN3)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="3" column="6" rowspan="3">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" rowspan="5">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1">
     <widget class="QPlainTextEdit" name="Text_status"/>
    </item>
    <item row="6" column="1" colspan="5">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="PB_Start">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Parse</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
